docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst

```diff
@@ -0,0 +1,124 @@
+Create a MQTT source connector for RabbitMQ®
+============================================
+
+In order to support the `MQTT source connector <https://docs.lenses.io/5.0/integrations/connectors/stream-reactor/sources/mqttsourceconnector/>`_ the `RabbitMQ MQTT plugin <https://www.rabbitmq.com/mqtt.html>`_ must be enabled. 
```

Comment:
I would start with a phrase on what the connector does before jumping to the details of the plugin

Reply:
done